src/main/scala/uk/gov/hmrc/initrepository/Github.scala

```diff
@@ -146,8 +154,44 @@ trait Github {
   def addRequireSignedCommits(repoName: String, branches: Seq[String]): Future[String] = {
     branches match {
       case Nil => Future.successful(s"Repo $repoName does not require signed commits")
-      case _   => Future.sequence(branches.map(addRequireSignedCommitsToBranch(repoName, _)))
-        .map(_.mkString(", "))
+      case _   => Future.sequence(branches.map { branch =>
+        for {
+          _      <- addBranchProtection(repoName, branch)
+          result <- addRequireSignedCommitsToBranch(repoName, branch)
+        } yield {
+          result
+        }
+      }).map(_.mkString(", "))
+    }
+  }
+
+  private def addBranchProtection(repoName: String, branch: String) = {
+    Log.info(s"Adding branch protection to repo $repoName for branch $branch")
+    val url = githubUrls.addBranchProtection(repoName, branch)
+
+    val payload = s"""{
+                     |    "required_status_checks": null,
```

Comment:
"Require status checks to pass before merging. Set to null to disable"
Does that mean you can merge before CI builds?